andela/firestar-backend
# configure eslint using airbnb base [finish #167730729]

#### What does this PR do?
- A summary of what this task is about
This PR configures eslint for the project
#### Description of Task to be completed?
- A list of the major tasks you did to complete this feature/bug/chore
 installed eslint eslint-airbnb-base eslint-plugin-import
 created the .eslintrc file and stated the rules there
#### How should this be manually tested?
- Outlined steps on how someone else can successfully test this feature/chore/bug to ensure it works as expected
- clone the repo to local machine [branch](https://github.com/Dolapo47/firestar-backend.git)
- run 'npm install' to install the necessary dependencies
- create a javascript file and test eslint on it 

#### What are the relevant pivotal tracker stories?
- [Pivot tracker link](https://www.pivotaltracker.com/story/show/167730729)
